engines/bops_core/app/controllers/concerns/bops_core/positions_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module BopsCore
+  module PositionsController
+    extend ActiveSupport::Concern
+
+    included do
+      wrap_parameters false
+
+      before_action :set_planning_application
+      before_action :set_collection
+      before_action :set_record
+    end
+
+    def update
+      respond_to do |format|
+        format.json do
```

Comment:
Ensure that we only ever deal with JSON responses and we'll return an "406 Not Acceptable" to things like bots and webcrawlers